### PR TITLE
New alerts

### PR DESCRIPTION
**Pull request by laurieboyes**
Go to `/myft/alerts-new` and gape in wonder. Not very interesting yet but worth getting in master anyway.

screen shot 2016-06-29 at 19 30 32

### CODE REVIEW

**Comment by keirog** [comment]
How do you feel about the route being the same for both, and changing on flag value?

**Comment by keirog** [comment]
Otherwise 👍

File: n.Makefile [hunk 1]
```diff
@@ -96,27 +96,32 @@ _install_scss_lint:
...
 # HACK: Use backticks rather than xargs because xargs swallow exit codes (everything becomes 1 and stoopidly scss-lint exits with 1 if warnings, 2 if errors)
 	@if [ -e .scss-lint.yml ]; then { scss-lint -c ./.scss-lint.yml `$(call GLOB,'*.scss')`; if [ $$? -ne 0 -a $$? -ne 1 ]; then exit 1; fi; $(DONE); } fi
 
 # DEPLOY SUB-TASKS
 
+APEX_PROD_ENV_FILE = .env.prod.json
 _deploy_apex:
-	@if [ -e project.json ]; then $(call CONFIG_VARS,production) | sed 's/\(.*\)/-s \1/' | tr '\n' ' ' | xargs apex deploy && $(DONE); fi
+	@if [ -e project.json ]; then $(call CONFIG_VARS,production) > $(APEX_PROD_ENV_FILE) && apex deploy --env-file $(APEX_PROD_ENV_FILE); fi
+	@if [ -e $(APEX_PROD_ENV_FILE) ]; then rm $(APEX_PROD_ENV_FILE) && $(DONE); fi
```
Comment (phamann) [inline n.Makefile]: Can we do `n-makefile` updates in separate PRs

File: views/alerts-new.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+			<div data-o-grid-colspan="12 M8 center">
+
+				<h2>My newsletters</h2>
+
+				{{#each userLists}}
+					<h3>{{list.name}}</h3>
+					<p>Sub level: {{list.subscriptionLevel}}</p>
+					<p>Image: ???</p>
+					<p>Description: ???</p>
+					<p>Frequency: ???</p>
```
Comment (phamann) [inline views/alerts-new.html]: Why have you omitted the above two data, they are available :)

Comment (laurieboyes) [inline views/alerts-new.html]: The long description and frequency aren't currently available. They're currently hard-coded in the old page, andrew's going to talk to the rest of the email platform team about getting them in the API for our purposes

**Pull request merged**